content/docs/command-reference/import.md

```diff
@@ -276,3 +278,133 @@ outs:
 
 The `url` and `rev_lock` subfields under `repo` are used to save the origin and
 [version](https://git-scm.com/docs/revisions) of the dependency, respectively.
+
+## Example: Chained imports
+
+DVC supports importing data importing data that was imported into the source
+repo, as long as all repos in the import chain are accessible from the final
+destination repo. Each source repo's default DVC remote must also be accessible
+from the final destination repo.
+
+Consider an example where we start with two DVC repos, A and B, with the
+following contents:
```

Comment:
Instead of starting with two repos, it may be better to start with one.

Reply:
I updated the whole example, PTAL @iesahin